client/js/libs/handlebars/formatNumber.js

```diff
@@ -0,0 +1,9 @@
+Handlebars.registerHelper(
+	"formatNumber", function(count) {
```

Comment:
Instead of the generic word "formatNumber" (that we might need somewhere else), shouldn't we call this something like, I don't know, "badgifyNumber", ... ?